Error rendering "index": No title found for chapter in JSON data

I tried updating mdbook to 0.3.2, but I am getting the following error:

```
$ mdbook build
2019-10-24 13:03:48 [INFO] (mdbook::book): Book building has started
2019-10-24 13:03:48 [INFO] (mdbook::book): Running the html backend
2019-10-24 13:03:48 [ERROR] (mdbook::utils): Error: Rendering failed
2019-10-24 13:03:48 [ERROR] (mdbook::utils):    Caused By: Error rendering "index" line 170, col 25: No title found for chapter in JSON data
```

What does this error mean? Is there some sort of JSON header we need to add to some files?

EDIT: Lol, forgot to mention that this is for the [rustc-guide](https://github.com/rust-lang/rustc-guide)

--- a/src/renderer/html_handlebars/helpers/navigation.rs
+++ b/src/renderer/html_handlebars/helpers/navigation.rs
@@ -69,7 +69,15 @@ fn find_chapter(
         // "index.md" (unless there really is an index.md in SUMMARY.md).
         match target {
             Target::Previous => return Ok(None),
-            Target::Next => match chapters.iter().skip(1).next() {
+            Target::Next => match chapters
+                .iter()
+                .filter(|chapter| {
+                    // Skip things like "spacer"
+                    chapter.contains_key("path")
+                })
+                .skip(1)
+                .next()
+            {
                 Some(chapter) => return Ok(Some(chapter.clone())),
                 None => return Ok(None),
             },

--- a/tests/dummy_book/src/SUMMARY.md
+++ b/tests/dummy_book/src/SUMMARY.md
@@ -1,6 +1,9 @@
 # Summary
 
 [Dummy Book](README.md)
+
+---
+
 [Introduction](intro.md)
 
 - [First Chapter](first/index.md)

--- a/tests/rendered_output.rs
+++ b/tests/rendered_output.rs
@@ -306,7 +306,7 @@ fn check_first_toc_level() {
 #[test]
 fn check_spacers() {
     let doc = root_index_html().unwrap();
-    let should_be = 1;
+    let should_be = 2;
 
     let num_spacers = doc
         .find(Class("chapter").descendant(Name("li").and(Class("spacer"))))

